lib/ddlog_std.dl

```diff
@@ -209,11 +209,8 @@ extern function default(): 'T
 
 /* Returns the contained `Some` value or a provided default.
  */
-function unwrap_or(x: Option<'A>, def: 'A): 'A = {
-    match (x) {
-        Some{v} -> v,
-        None -> def
-    }
+function unwrap_or(#[by_val] x: Option<'A>, def: 'A): 'A = {
```

Comment:
Why not make both params by value?

Reply:
It's probably going to make less of a difference, but there's no real downside either.  The main concern with making something `#[by_val]` is that the called may then need to clone if they only have access to the value by reference.  Here this should not be an issue.